packages/report-server/src/types.ts

```diff
@@ -30,6 +30,7 @@ export interface ReportConfig {
 interface ReportsRequestBody {
   testConfig?: ReportConfig;
   testData?: Record<string, string | number>[];
+  queryOverride?: FetchReportQuery;
```

Comment:
I think I just had a chat with Edwin about this and he mentioned he might be thinking of changing this, but I feel it might be a little nicer if the query options were just present on the root level of this type rather than nested with the `queryOverride` field?

For what it's worth the plan is to move the `testConfig` and `testData` fields out sometime soon, I wanna create an entirely separate route for the 'test' logic